<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieViewerDialogSub</class>
 <widget class="QWidget" name="MovieViewerDialogSub">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="33,12,47">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="CustomLabel" name="indexLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CustomLabel" name="laserSpeedLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CustomLabel" name="laserDistanceLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CustomLabel" name="dateLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CustomLabel" name="timeLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CustomLabel" name="rtLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="6">
        <widget class="CustomPushButton" name="printPushButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="CustomPushButton" name="twoSpeedPushButton">
         <property name="text">
          <string>2x</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="CustomPushButton" name="halfspeedPushButton">
         <property name="text">
          <string>0.5x</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="CustomPushButton" name="quitPushButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="CustomPushButton" name="previousPushButton">
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="CustomPushButton" name="nextPushButton">
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="CustomPushButton" name="playPushButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomPushButton</class>
   <extends>QPushButton</extends>
   <header>CustomPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header>CustomLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
